SpatialGDK/Source/SpatialGDK/Private/SpatialView/ViewDelta.cpp

```diff
@@ -139,6 +139,11 @@ const TArray<EntityDelta>& ViewDelta::GetEntityDeltas() const
 	return EntityDeltas;
 }
 
+TArray<Worker_Op>& ViewDelta::GetWorkerMessages()
```

Comment:
I do not see this non-const version used anywhere?

Reply:
ah this was a remnant of sami's work and he hadn't removed it. will remove it